config/sample_config.json

```diff
@@ -0,0 +1,60 @@
+{
+  "default_auth_info":
+  {
+    "auth_type": "passport",
+    "auth_token": "passport_token_2.txt"
+  },
+  "service_info_auth":
+  {
+    "auth_type": "none",
+    "auth_token": ""
+  },
+  "drs_objects":
+  [
+    {
+      "drs_id": "697907bf-d5bd-433e-aac2-1747f1faf366"
+    },
+    {
+      "drs_id": "1af6b862-7fc8-411a-86c5-d8e280e5b77a",
+      "auth_info":
+      {
+        "auth_type": "basic",
+        "auth_token": "basic_token_1.txt"
+      },
+      "expected_status_code": 400
+    },
+    {
+      "drs_id": "41898242-62a9-4129-9a2c-5a4e8f5f0afb",
+      "auth_info":
+      {
+        "auth_type": "bearer",
+        "auth_token": "bearer_token_1.txt"
+      },
+      "has_auth_get_object": true,
+      "expected_status_code": 200
+    },
+    {
+      "drs_id": "a1dd4ae2-8d26-43b0-a199-342b64c7dff6",
+      "auth_info":
+      {
+        "auth_type": "passport",
+        "auth_token":
+        [
+          "passport_token_1.txt",
+          "passport_token_2.txt"
+        ]
+      },
+      "is_bundle": true,
```

Comment:
I know that this compliance suite is for 1.2.0, but we know that bundles are going to be deprecated. I'm not sure if it's necessary to include a bundle in the test schema as most people's DRS implementation will not have the infrastructure to handle this.